packages/components/src/IconsProvider/__snapshots__/IconsProvider.test.js.snap

```diff
@@ -6154,7 +6155,8 @@ exports[`IconsProvider should render default talend-icons and custom icons defin
+          d="M9.0107,15.082 C5.5477,15.082 2.7307,12.295 2.7307,8.868 C2.7307,8.281 2.8127,7.71 2.9777,7.168 C3.7327,4.505 6.2137,2.645 9.0107,2.645 C12.4687,2.645 15.2817,5.436 15.2817,8.868 C15.2817,12.295 12.4687,15.082 9.0107,15.082 M9.2607,1.914 L9.2607,1.186 L3.9747,1.186 L2.8737,-1.77635684e-15 L-0.0003,-1.77635684e-15 L-0.0003,7.35 L2.1667,7.35 C2.0517,7.846 1.9927,8.355 1.9927,8.868 C1.9927,12.7 5.1407,15.817 9.0107,15.817 C12.8747,15.817 16.0187,12.7 16.0187,8.868 C16.0187,5.114 13.0097,2.046 9.2607,1.914 M11.5707,8.1581 C11.0597,8.1581 10.6107,7.9401 10.2767,7.6001 L8.5547,8.5201 C8.5857,8.6651 8.6067,8.8091 8.6067,8.9541 C8.6067,9.0481 8.5967,9.1311 8.5857,9.2241 L10.3607,10.1751 C10.6837,9.8861 11.1007,9.6991 11.5707,9.6991 C12.5627,9.6991 13.3557,10.4961 13.3557,11.4681 C13.3557,12.4401 12.5627,13.2371 11.5707,13.2371 C10.5897,13.2371 9.7857,12.4401 9.7857,11.4681 C9.7857,11.3441 9.8067,11.2301 9.8277,11.1161 L8.1577,10.2171 C7.7827,10.6921 7.1987,10.9921 6.5517,10.9921 C5.4137,10.9921 4.4947,10.0711 4.4947,8.9541 C4.4947,7.9821 5.1737,7.1651 6.0917,6.9581 C6.2377,6.9271 6.3947,6.9061 6.5517,6.9061 C6.7077,6.9061 6.8647,6.9271 7.0107,6.9581 C7.4067,7.0521 7.7717,7.2681 8.0437,7.5581 L9.8177,6.6171 C9.8067,6.5441 9.7857,6.4621 9.7857,6.3901 C9.7857,5.4061 10.5897,4.6201 11.5707,4.6201 C12.5627,4.6201 13.3557,5.4061 13.3557,6.3901 C13.3557,7.3611 12.5627,8.1581 11.5707,8.1581 Z M8.0537,8.9541 L8.0537,8.7891 C8.0017,8.4061 7.8137,8.0651 7.5427,7.8271 C7.2817,7.5891 6.9267,7.4551 6.5517,7.4551 C5.7167,7.4551 5.0377,8.1271 5.0377,8.9541 C5.0377,9.7821 5.7167,10.4441 6.5517,10.4441 C6.9897,10.4441 7.3857,10.2581 7.6577,9.9581 C7.9077,9.6991 8.0537,9.3481 8.0537,8.9541 Z"
+          fill="#010202"
```

Comment:
all other icons use black ass default fill are we sure we want to use this ?

Reply:
you're right i don't know why the color turns to #010202 instead of #202020. But it's not very impacting our future design because it will use $Dove gray and $Brand primary in the design